src/core/render-core.js

```diff
@@ -150,6 +165,10 @@ async function render(_opts = {}) {
 
       data = await page.screenshot(screenshotOpts);
     }
+
+    // Write data to file
+    logger.info('[renderCore] write to file, jobId = ', opts.jobId);
+    await writeToFile(`/home/pdf/${opts.jobId}.pdf`, data);
```

Comment:
Is this the path in hdfs?

Reply:
Yes, this is a temporary location for pdf file. Do we have a permanent location? Or do we want to pass in the location in post body?

Reply:
I agree that the location on HDFS should be parameterized and managed by ansible. This way we can make sure the directory with proper permissions is created before running the service. The path can then be returned when done writing to HDFS, but is not required to be provided in the post body